templates/static/js/example.js

```diff
@@ -293,4 +307,5 @@
       }
     });
   }
+  fixScroll();
```

Comment:
Around line 160+ we have a section that does resizing of the iframes after they load.
I guess this fixScroll() could be called in there? So it triggers after iframe loads, not when page loads.

Reply:
Thanks - that seems to work a treat